util/reggen/reg_block.py

```diff
@@ -382,9 +382,16 @@ def make_intr_regs(self, interrupts: Sequence[Signal]) -> None:
                                  'rw1c',
                                  'hrw',
                                  False,
-                                 # intr_state csr is affected by writes to
-                                 # other csrs - skip write-check
-                                 ["excl:CsrNonInitTests:CsrExclWriteCheck"])
+                                 # see issue#6888 for further details.
+                                 #
+                                 # The rom_ctrl<->kmac interaction right after
+                                 # reset causes kmac.intr_state to become
+                                 # updated before CSR tests can start at the
+                                 # chip leve, leading to unexpected failures.
+                                 #
+                                 # As per #6888, exclude intr_state from CSR
+                                 # checks to avoid this issue.
+                                 ["excl:CsrAllTests:CsrExclAll"])
```

Comment:
Should we write-exclude the `alert_test` register below?

Reply:
I think in csr_rw we disable `alert_en` register in alert_handler, so even though alert_test is set to 1, it will trigger an alert, but won't cause any escalation in the system.

Reply:
At the chip level, sure. But at the IP level writing to this CSR will cause alert to trigger, right? I suppose that is inconsequential if the scoreboard is disabled....so we are probably good to go without the exclusion.

Reply:
Its fine to not address this. Lets get it merged so that the CI issue is resolved.